internal/config/sources.go

```diff
@@ -162,6 +163,26 @@ func GetActiveSourceByType(sources []Sources, key string) []Sources {
 	return sourcesSlice
 }
 
+func GetActiveSourceByHeadless(sources []Sources, headless bool) []Sources {
```

Comment:
;)

Reply:
It just seemed easier than re-checking them every time, heh.